MacDown/Code/Preferences/MPTerminalPreferencesViewController.m

```diff
@@ -152,8 +152,8 @@ - (void)installShellUtility
     NSFileManager *fm = [NSFileManager defaultManager];
     if ([fm fileExistsAtPath:utilityBundlePath])
     {
-        BOOL ok = [fm copyItemAtPath:utilityBundlePath
-                              toPath:MPCommandInstallationPath error:NULL];
+        BOOL ok = [fm createSymbolicLinkAtPath:MPCommandInstallationPath
+                           withDestinationPath:utilityBundlePath error:NULL];
```

Comment:
Perfect. I assumed symlinking wasn't possible through `NSFileManager` but happy to be wrong